### PR TITLE
CVE-2019-14881 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14881.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14881",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14881",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14881.json]: Please add a reference to upstream security advisory:

 https://moodle.org/mod/forum/discuss.php?d=393584#p1586746

File: 2019/14xxx/CVE-2019-14881.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in moodle 3.7 to 3.7.2 where there is blind XSS reflected in some locations where user email is displayed."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14881.json]: Please change the description to mention fixed version, i.e. 3.7.3.

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14881.json [hunk 3]
```diff
@@ -59,7 +59,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "A vulnerability was found in moodle through 3.7 to 3.7.2 and before 3.7.3, where there is blind XSS reflected in some locations where user email is displayed."
+                "value": "A vulnerability was found in moodle 3.7 to 3.7.2 and before 3.7.3, where there is blind XSS reflected in some locations where user email is displayed."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14881.json]: It seems there is a copypasto.
Please change from:

````
[...] 3.7 to 3.7.2 and before 3.7.3 [...]
````
````
[...] versions 3.7 before 3.7.3 [...]
````

**Comment by iamleot** [comment]
@cve-team Can you please review possible comments and commits before merging?  Thanks!

**Pull request merged**